### ISSUE TITLE
Users other than admin unable to login

### ISSUE THREAD

**Issue by jairana** [elmsln/elmsln#789]
I am unable to login other than "admin". So only user is working is admin.
I have installed Elmsln many time but the same problem repeat again and again.
I created new users after login default account admin and make them administrator. Then logout the admin and try to login as new user created but unable to  login. 
Even if i try to change username of admin and password of admin it does not change at all.
If i create new account on login screen, account is created, the user  login in elmsln, using admin i unblocked and make this administrator. But second time if i login using this user account (created on login screen). It does not login.

So in short i am able to create new account but unable to login 

I have installed latest  elmsln on digitalocean with one line command. i followed your Youtube videos created on 10 march, 16.

**Comment by btopro** [comment]
If you go to the people domain (snowflake, then people) and make the account there it'll be able to show up across them all (or should). This is an issue resolved in university deployments via other methods and something we're working on getting in associated with the #617

**Comment by jairana** [comment]
Bakery setting worked for me.
I make one site master site and other slave sites.
Now i am able to login with other user names
Thanks
